lib/trader.js

```diff
@@ -64,8 +64,17 @@ Trader.prototype.syncLogs = function syncLogs () {
     .then(data => data.body)
   // Delete log files that are two or more days old
     .then(() => {
```

Comment:
We need to pass data.body to logs.queryNewestLogs, name the param here.

```
.then((it) => {
```

Reply:
Note that you can also refactor this delete stuff to a separate function and call it outside this promise chain, just make sure to add error handling to it.